gulpfile.js

```diff
@@ -81,6 +83,22 @@ var DEFINES = {
   PDFJS_NEXT: false,
 };
 
+function safeSpawnSync(command, parameters, options) {
+  // Windows has problems executing commands outside a shell.
+  if (process.platform === 'win32') {
```

Comment:
do we need it for all commands or for some?
can we add shell option for certain command on all platforms?